test/verify/check-storage-partitions

```diff
@@ -61,6 +61,40 @@ class TestStorage(StorageCase):
         self.confirm()
         self.content_row_wait_in_col(1, 1, "Free Space")
 
+    def testSizeSlider(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+        b.wait_in_text("#drives", "VirtIO")
+
+        m.add_disk("50M", serial="DISK1")
+        b.wait_in_text("#drives", "DISK1")
+        b.click('#drives tr:contains("DISK1")')
+        b.wait_present('#storage-detail')
+
+        b.wait_visible('button:contains(Create partition table)')
+        b.click('button:contains(Create partition table)')
+        self.dialog({"type": "gpt"})
+        self.content_row_wait_in_col(1, 1, "Free Space")
+
+        self.content_row_action(1, "Create Partition")
+        self.dialog_wait_open()
+        self.dialog_set_val("type", "empty")
+
+        slider = self.dialog_field("size") + " .slider"
+
+        # Move the slider one pixel past the middle, this should give a fractional size.
+        # See https://github.com/cockpit-project/cockpit/pull/10968 for more about this.
+        width = b.call_js_func('(function (sel) { return ph_find(sel).offsetWidth; })', slider)
+        about_half_way = width / 2 + 1
+        b.mouse(slider, "mousedown", about_half_way, 0)
+        b.mouse(slider, "mouseup", about_half_way, 0)
```

Comment:
This might be a  potential race condition here? Should this wait until the input line shows "25"? With the rounding this should be predictable.

Reply:
How can this be a race?

Reply:
I thought because we don't wait for the event handlers to get processed. It would also be nice to verify that the input line actually reacts to the slider.